pkg/controller/jobframework/reconciler.go

```diff
@@ -417,13 +417,9 @@ func (r *JobReconciler) ReconcileGenericJob(ctx context.Context, req ctrl.Reques
 		if workload.HasQuotaReservation(wl) {
 			if !job.IsActive() {
 				log.V(6).Info("The job is no longer active, clear the workloads admission")
-				if evCond.Reason == kueue.WorkloadEvictedByDeactivation ||
-					evCond.Reason == kueue.WorkloadEvictedByPodsReadyTimeout ||
-					evCond.Reason == kueue.WorkloadEvictedByClusterQueueStopped {
-					workload.SetRequeuedCondition(wl, evCond.Reason, evCond.Message, false)
-				} else {
-					workload.SetRequeuedCondition(wl, evCond.Reason, evCond.Message, true)
-				}
+				// The requeued condition status set to true only on EvictedByPreemption or EvictedByAdmissionCheck
+				status := evCond.Reason == kueue.WorkloadEvictedByPreemption || evCond.Reason == kueue.WorkloadEvictedByAdmissionCheck
```

Comment:
Rename `status` to something like `setRequeue` or anything more relevant.

Reply:
Done